### PR TITLE
Provides a custom component to a BOS Custom cookie prompt to replace the iubenda plugin

### PR DESCRIPTION
**Pull request by charleslavon**

### CODE REVIEW

**Comment by shelegdmitriy** [comment]
Seems like there is some PR on the components repo to test it fully, right?

**Comment by charleslavon** [comment]
> Seems like there is some PR on the components repo to test it fully, right?

[it's merged](https://github.com/near/near-discovery-components/pull/664)

**Review by calebjacob — APPROVED** [review]

File: src/utils/types.ts [hunk 1]
```diff
@@ -33,7 +33,6 @@ type ProductionNetwork = {
 };
 
 export interface TosData {
-  showTos: boolean;
```
Comment (shelegdmitriy) [inline src/utils/types.ts:36]: Why you decided to remove this?

Comment (charleslavon) [reply src/utils/types.ts:36]: There were no references to it.

**Pull request merged**